lib/core/src/Cardano/Wallet/Api/Types.hs

```diff
@@ -1238,7 +1257,8 @@ data ApiDecodedTransaction (n :: NetworkDiscriminant) = ApiDecodedTransaction
 
 -- | The response cardano-wallet returns upon successful submission of a
 -- mint/burn transaction.
-data ApiMintedBurnedTransaction (n :: NetworkDiscriminant) = ApiMintedBurnedTransaction
+data ApiMintedBurnedTransaction (n :: NetworkDiscriminant) =
```

Comment:
```suggestion
data ApiMintBurnTransaction (n :: NetworkDiscriminant) =
```

Reply:
this one is not used in new tx workflow and should be removed/cleaned in coming PR